### PR TITLE
cleaning up routing structure

### PR DESCRIPTION
**Pull request by rossem**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,10 @@ erl_crash.dump
...
 /config/prod.secret.exs
 
-#Swaps
+# Vim
 *.swo
 *.swp
+
+# Emacs
+\#*\#
+.\#*
```
Comment (nare469) [inline .gitignore]: whoa buddy

Comment (rossem) [inline .gitignore]: vuck fim

File: web/controllers/user_controller.ex [hunk 2]
```diff
@@ -3,21 +3,16 @@ defmodule Api.UserController do
 
   alias Api.User
 
-  plug :scrub_params, "user" when action in [:create, :update]
+  plug :scrub_params, "user" when action in [:createUser, :updateUser]
```
Comment (nare469) [inline web/controllers/user_controller.ex]: Any reason you suffixed with `User`?

Comment (rossem) [inline web/controllers/user_controller.ex]: just to be explicit, i thought of just making them crud, but what if we want to add like more shit. `findUserFiles` is better than `findFiles`

Comment (nare469) [inline web/controllers/user_controller.ex]: I fell like it's always going to be `User.findFiles` or something like that when it's being referenced from somewhere else.

Comment (rossem) [inline web/controllers/user_controller.ex]: but its just controller name for specific route, won't get reused really

Comment (nare469) [inline web/controllers/user_controller.ex]: Hmm. I guess it works. :bear:

**Comment by Wojtechnology** [comment]
:bear: :+1: :+1: :+1:

**Pull request merged**